dev/tests/api-functional/framework/Magento/TestFramework/TestCase/GraphQl/ResponseContainsErrorsException.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\TestFramework\TestCase\GraphQl;
+
+/**
+ * Response contains errors exception
+ */
+class ResponseContainsErrorsException extends \Exception
+{
+    /**
+     * @var array
+     */
+    private $responseData;
+
+    /**
+     * @param string $message
+     * @param array $responseData
+     * @param \Exception|null $cause
+     * @param int $code
+     */
+    public function __construct(string $message, array $responseData, \Exception $cause = null, int $code = 0)
```

Comment:
We should not swap the original arguments order to comply with Liskov substitution principle.